src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -0,0 +1,475 @@
+swagger: '2.0'
+info:
+  description: |
+    **Model API**
+
+    Provides access to the model features of Alfresco Content Services.
+  version: '1'
+  title: Alfresco Content Services REST API
+basePath: /alfresco/api/-default-/public/alfresco/versions/1
+securityDefinitions:
+  basicAuth:
+    type: basic
+    description: HTTP Basic Authentication
+security:
+  - basicAuth: []
+consumes:
+  - application/json
+produces:
+  - application/json
+tags:
+  - name: aspects
+    description: Retrieve and manage aspects
+  - name: types
+    description: Retrieve and manage types
+
+paths:
+  '/aspects':
+    get:
+      x-alfresco-since: "7.0.0"
+      tags:
+        - aspects
+      summary: List aspects
+      description: |
+        **Note:** This is available in Alfresco 7.0.0 and newer versions.
+
+        Gets a list of aspects from the data dictionary. The System aspects will be ignored by default.
+        ```JSON
+        {
+          "list": {
+            "pagination": {
+              "count": 0,
+              "hasMoreItems": true,
+              "totalItems": 0,
+              "skipCount": 0,
+              "maxItems": 0
+            },
+            "entries": [
+              {
+                "entry": {
+                  "id": "cm:titled",
+                  "description": "Titled",
+                  "title": "Titled",
+                  "properties": [
+                    {
+                      "id": "cm:title",
+                      "title": "Title",
+                      "description": "Content Title",
+                      "dataType": "d:mltext",
+                      "isMultiValued": false,
+                      "isMandatory": false,
+                      "isMandatoryEnforced": false
+                      "isProtected": false
+                    },
+                    {
+                      ...
+                    }
+                  ]
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+            ]
+          }
+        }
+        ```
+      operationId: listAspects
+      parameters:
+        - name: where
+          in: query
+          description: |
+            Optionally filter the list. Here are some examples:
+
+            An aspect should represented in the fully qualified name(`namespace_prefix:name`). e.g 'cm:title'.
```

Comment:
Note: doesn't "fully-qualified" means using {uri}localName rather than prefix:localName format ?

In any case, do we need a /namespaces API to list and map between these two formats. Or maybe that will be possible in future /models API ?

Reply:
can you add this to RFC ? we can have a  discussion